components/Dialogs/src/MDCAlertController.m

```diff
@@ -212,6 +212,11 @@ - (void)setButtonTitleColor:(UIColor *)buttonColor {
   }
 }
 
+- (void)setTitleAlignment:(NSTextAlignment)titleAlignment {
+  _titleAlignment = titleAlignment;
+  self.alertView.titleAlignment = titleAlignment;
```

Comment:
It is not required, but consider adding if (self.alertView) to be consistent with the other properties and add context for the next engineer looking at this code.